gcp/compute_engine/resource_policies/disk_snapshot_schedule/vars.tf

```diff
@@ -0,0 +1,67 @@
+variable "name" {
+  type = string
+  default = "snapshot-schedule"
```

Comment:
Needs something to make the default unique?

Reply:
Does this need to be unique? Can add date-time to the end if necessary, but it seems like the name of the snapshot policy has no effect on name of the snapshot

Reply:
Just thinking if 2 policies are created and don't specify a name but now I think about it we probably should enforce this to be given.

Reply:
Default for `name` now removed